Resources/public/js/views/ez-usermenuview.js

```diff
@@ -0,0 +1,225 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-usermenuview', function (Y) {
+    'use strict';
+    /**
+     * Provides the User Menu View class
+     *
+     * @module ez-usermenuview
+     */
+    Y.namespace('eZ');
+
+    var CLASS_HIDDEN = 'is-user-menu-hidden',
+        CLASS_USER_PROFILE = 'ez-user-profile',
+        SELECTOR_USER_PROFILE = '.' + CLASS_USER_PROFILE,
+        SELECTOR_USER_MENU = '.ez-user-menu';
+
+    /**
+     * The User Menu view
+     *
+     * @namespace eZ
+     * @class UserMenuView
+     * @constructor
+     * @extends eZ.TemplateBasedView
+     */
+    Y.eZ.UserMenuView = Y.Base.create('userMenuView', Y.eZ.TemplateBasedView, [], {
+        events: {
+            '.ez-user-menu-item': {
+                'tap': '_toggleUserMenu'
+            },
+            '.ez-user-menu-item-logout': {
+                'tap': '_logOut'
+            }
+        },
+
+        initializer: function () {
+            this.on('*:toggleUserMenu', this._toggleUserMenu);
+        },
+
+        /**
+         * Renders the user menu view
+         *
+         * @method render
+         * @return {eZ.UserMenuView} the view itself
+         */
+        render: function () {
+            var container = this.get('container');
+
+            container.setHTML(this.template(this.getAttrs()));
+            container.on('clickoutside', this._hideMenuOnClickOutside, this);
```

Comment:
you should store the subscription object (the return value of `on`) to make sure the subscription is detached when the view is destroyed.